doc/source/whatsnew/v2.2.0.rst

```diff
@@ -433,8 +425,6 @@ Optional libraries below the lowest tested version may still work, but are not c
 +=================+=================+=========+
```

Comment:
Could we just remove this table and recommend the users check the ``` :ref:`install.optional_dependencies` ``` for the new minimum versions tested?

Reply:
Would be hard to figure out what changed if you really care about those versions

that said, probably nobody really does, so maybe just move that table all the way to the bottom of the whatsnew?

Reply:
That's fair. I think I updated these version a while back so I'll make a follow up to just have one table with the updated verions